data/evidence/allegation.go

```diff
@@ -45,6 +46,32 @@ type AllegationTracker struct {
 	Requests map[string]bool
 }
 
+func (es *EvidenceStore) CleanTracker() {
+	at, err := es.GetAllegationTracker()
+	if err != nil {
+		return
+	}
+	requestIdList := make([]string, len(at.Requests))
+	for k := range at.Requests {
+		requestIdList = append(requestIdList, k)
+	}
+	sort.Strings(requestIdList)
+	countMap := make(map[string]bool)
+	for _, r := range requestIdList {
+		ar, err := es.GetAllegationRequest(r)
+		if err != nil {
+			delete(at.Requests, r)
```

Comment:
What scenario is this delete for?

Reply:
This is a fail-safe , in case we have a request-Id , which does not have an associated allegation request